### PR TITLE
getDownloadURL is updated

### PR DESCRIPTION
**Pull request by KarthikShanmugasundaram**

### CODE REVIEW

File: docs/files.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ file's contents, upload new versions, and perform other common file operations
 * [Update a File's Information](#update-a-files-information)
 * [Get a File's Tasks](#get-a-files-tasks)
 * [Download a File](#download-a-file)
+* [Get a File's download URL](#get-download-url)
```
Comment (mattwiller) [inline docs/files.md]: [nit] Capitalize "download" for consistency

File: docs/files.md [hunk 2]
```diff
@@ -106,6 +107,24 @@ client.files.getReadStream('12345', null, function(error, stream) {
 });
 ```
 
+Get a File's download URL
```
Comment (mattwiller) [inline docs/files.md]: [nit] Capitalize "download" for consistency

File: docs/files.md (hunk 1 above)
Comment (KarthikShanmugasundaram) [inline docs/files.md]: Done. Thank you..

File: docs/files.md (hunk 2 above)
Comment (KarthikShanmugasundaram) [inline docs/files.md]: done. Thank you..

**Pull request merged**